config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
+
+<module name="Checker">
+    <module name="JavadocPackage"/>
+    <module name="TreeWalker">
+        <module name="AvoidStarImport"/>
```

Comment:
Good for start but consider to add more rules

Reply:
I has added more modules and refactored their warnings